ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installation-nightly-4.20.yaml

```diff
@@ -454,6 +454,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       ENABLE_REBOOT_CHECK: "true"
+      REBOOT_TYPE: HARD_REBOOT
```

Comment:
I remember we have aws-ipi-reboot job, we need to config HARD_REBOOT for it as well.